blockbuster/src/programs/bubblegum/mod.rs

```diff
@@ -195,18 +197,15 @@ impl ProgramParser for BubblegumParser {
                             Some(build_creator_verification_payload(keys, ix_data, true)?);
                     }
                     InstructionName::UnverifyCreator => {
-                        let payload = build_creator_verification_payload(keys, ix_data, false)?;
-                        b_inst.payload = Some(payload);
+                        b_inst.payload =
+                            Some(build_creator_verification_payload(keys, ix_data, false)?);
                     }
-                    // We don't extract any additional info w.r.t. verify and unverify
-                    // collection ops for now.
-                    InstructionName::SetAndVerifyCollection => {
-                        // Deserializing this to get to the second argument encoded in the slice,
-                        // which is the collection address. Is there a (safe) way to get to that
-                        // directly?
-                        let _args: MetadataArgs = MetadataArgs::try_from_slice(ix_data)?;
-                        let collection: Pubkey = Pubkey::try_from_slice(ix_data)?;
```

Comment:
Yes this never had the right offset for `MetadataArgs` or `collection`, which you figured out, and yesterday I found that `SetAndVerify` doesn't pass this parsing on mainline at the moment.